tests/ui/proc-macro/meta-macro-hygiene.rs

```diff
@@ -3,8 +3,10 @@
 // edition:2018
 // compile-flags: -Z span-debug -Z macro-backtrace -Z unpretty=expanded,hygiene -Z trim-diagnostic-paths=no
 // check-pass
+// ignore-tidy-linelength
```

Comment:
Are these and similar changes unrelated to this PR? If so, can you move to separate PR?

Reply:
The tests don't pass without these changes: there's more detail in https://github.com/rust-lang/rust/pull/111891/commits/4182a0e00e0e095ecbf598223001990c2777e1d5 but the upshot is that a previous change introduced an issue where these two files need to change any time a symbol is added. This is that change, and removes the coupling for the future.